doc/Makefile

```diff
@@ -29,7 +29,7 @@ pip3:
 	$(PIP_INSTALL) -r ../taskcluster/docs-requirements.txt
 
 npm:
-	npm install jsdoc@3.6.3
+	npm install typedoc@0.17.4 typescript@3.8.3 typedoc@0.17.4 electron@^1.7.9 node-gyp@5.x @types/argparse@1.0.x @types/node@13.9.x
```

Comment:
this should be part of `devDeps` no?

Reply:
As far as I can tell that doesn't affect the docs build process, hence why `jsdoc` was here before.

Reply:
The reason why this can't just be `typedoc` is because it needs to know the types for the packages used by `index.ts` as well. Although now that I think about it, this should not look at `client.ts`, so I can probably remove `argparse` and `electron`.